cognitedata/reveal
# feat(react-components): create and delete observations

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/BND3D-4179
https://cognitedata.atlassian.net/browse/BND3D-4180

(Note that the above tasks should also have more involved UI for observation handling, which is not addressed by this PR

## Description :pencil:


Buttons to create and delete observations, and committing the changes to the cloud.

## How has this been tested? :mag:



In storybook and Search/Fusion


## Test instructions :information_source:



- Go to storybook
- Connect to 3d-test
- Use `{ 
    modelId: 3544114490298106,
    revisionId: 6405404576933316 }` (you can change directly from id `-1`, `-1`in `getAddModelOptionsFromUrl.ts`)
- Activate observations by clicking the "paper plane-button" to the right in the Architecture story.
- Click "fit to view" to see the observations if they're not found.
- Click on the plus-button in the bottom-left toolbar
- Click somewhere on the model to create an observation
- Perhaps create a few more
- Click blue "save" button to save to cloud
- See that "save" disappears if nothing has been changed
- Refresh to see that observations persist
- Click created observations and the delete-button/ delete-key to mark for deletion
- Click "save" to see that it disappears
- Refresh to see that it persists

## Checklist :ballot_box_with_check:
